apps/namada-interface/src/App/AccountOverview/DerivedAccounts/DerivedAccounts.tsx

```diff
@@ -38,8 +37,6 @@ import { CoinsState, fetchConversionRates } from "slices/coins";
 import { SettingsState } from "slices/settings";
 import Config from "config";
 
-const { NAMADA_INTERFACE_NAMADA_FAUCET_ADDRESS: faucetAddress } = process.env;
```

Comment:
This could also be dropped from `.env.sample` now.

Reply:
Done!